irods/test/query_test.py

```diff
@@ -7,7 +7,7 @@
 import unittest
 import time
 import uuid
-from datetime import datetime
+from datetime import (datetime, timezone)
```

Comment:
why the parens here?  is that the new thing?

Reply:
oh, it's for long lines... 

https://stackoverflow.com/questions/29534027/why-import-statements-with-parentheses

Reply:
Yes, strictly optional for short ones.